jupyterhub/templates/hub/configmap.yaml

```diff
@@ -21,3 +21,6 @@ data:
 {{- $_ := set $values.singleuser "imagePullSecret" (omit .Values.singleuser.imagePullSecret "password") }}
   values.yaml: |
     {{- $values | toYaml | nindent 4 }}
+{{- range $key, $value := .Values.hub.extraConfigMap }}
+  custom.{{ $key }}: {{ toJson $value | quote }}
+{{- end }}
```

Comment:
This is the outcome of the new configmap.

```
apiVersion: v1
data:
  custom.callback-url: '"https://example.com/hub/oauth_callback"'
  custom.client-id: '"redacted"'
  custom.client-secret: '"redacted"'
  custom.hosted-domain: '"example.com"'
  custom.login-service: '"Example"'
  values.yaml: |
    auth:
      admin:
        access: false
        users:
# ...
```

I guess that the new get_config function looks in values.yaml, because I still got the same error

Reply:
ah, right. Need to preserve `.get_config`.